### DISCUSSION TITLE
How do you handle cross-platform development?

### DISCUSSION THREAD

**Discussion by ranolf** [MobileNativeFoundation/discussions#181] [Polls]
I am hoping to gather some data about industry practice.

At Intuit, most of our applications are built purely natively (mostly without SDUI), with some proprietary embedded web cases. 
One flavor of approaching cross-platform SDUI is to leverage a common UI layer across platforms, and I was hoping to get a sense of current industry practices.

**Comment by opnsrcntrbtr** [comment]
From my dev experiences across app business use cases, I recommend that native is the best approach if app requires latest features offered by Android and iOS native frameworks. When apps are merely clones of webapps in a mobile form factor with basic mobila app exclusive features like push notifications, etc then, and web app is developed using React or Angular, then pairing it with capacitor is preferable. React native is preferable if business logic can be shared with React web app and app features need to be performant and may lag latest features introduced in Android and iOS native frameworks. Flutter is preferable when web app shares no similarities with mobile app and a performant cross platform ap, and maintaining single code base cuts costs without compromising on feature sets like those on Android and iOS native frameworks, and team is expert on using Dart. 
Kindly share your experiences in various business use cases.
